pkg/accounting/accounting.go

```diff
@@ -108,10 +108,6 @@ func NewAccounting(
 		return nil, fmt.Errorf("tolerance plus threshold too big: %w", ErrOverflow)
 	}
 
-	if PaymentTolerance > PaymentThreshold/2 {
-		return nil, ErrInvalidPaymentTolerance
```

Comment:
is this error still used somewhere else? If not should probably remove it on top

Reply:
removed